validator/rpc/standard_api.go

```diff
@@ -382,3 +388,117 @@ func groupDeleteRemoteKeysErrors(req *ethpbservice.DeleteRemoteKeysRequest, erro
 	}
 	return statuses
 }
+
+// ListFeeRecipientByPubkey returns the public key to eth address mapping object to the end user.
+func (s *Server) ListFeeRecipientByPubkey(ctx context.Context, req *ethpbservice.ByPubkeyRequest) (*ethpbservice.GetFeeRecipientByPubkeyResponse, error) {
+	if s.validatorService == nil {
+		return nil, status.Error(codes.FailedPrecondition, "Validator service not ready.")
+	}
+	validatorKey := req.Pubkey
+	if len(validatorKey) != fieldparams.BLSPubkeyLength {
+		return nil, status.Errorf(
+			codes.InvalidArgument, "provided public key in path is not a valid bls public key, please check for correct hex format starting with 0x")
```

Comment:
Technically, it should only be 0x prefixed hex if this is called via the API middleware. If called via grpc, it needs to be raw bytes. However, no one will be calling this through gRPC I guess, unless it is done programmatically. Just to be safe, I would amend the error message to say the key needs to have the right length and and printf the field params value instead of telling the user it needs to be a 0x prefixed hex string